src/main/java/echoquery/intents/AggregationHandler.java

```diff
@@ -34,12 +34,6 @@ public SpeechletResponse respond(Intent intent, Session session) {
    * Exposed for testing purposes - SpeechletResponse is impossible to inspect.
    */
   public String getResponseInEnglish(Intent intent, Session session) {
-    try {
-      QueryResult result = querier.execute(QueryRequest.of(intent));
-      return result.toEnglish();
-    } catch (SQLException e) {
```

Comment:
I will miss runtime exceptions being an integral component of our basic error handling